### ISSUE TITLE
Add possibility to revoke / delete Site and API keys

### ISSUE THREAD

**Issue by BenjaminVettori** [FriendlyCaptcha/friendly-challenge#77]
I was searching for an option to revoke or delete a Site or API key in the dashboard but did not find a way to do that.
Did i just miss some option or is this currently really not possible?

For site keys it might not be important, but for API keys it is important in my opinion.
If an API key leaks for whatever reason, you need an option to replace it with a new key while revoking or even deleting the old one.

**Comment by gzuidhof** [comment]
Hi Benjamin,

It's not currently possible as a user in the interface, so it would involve making a ticket and we can remove it on our end. The api keys are not very destructive fortunately (i.e. they don't offer any other privileges than verifying puzzles), so the delay in us helping you do that shouldn't be too much of an issue.

We're shipping a new user interface this month or the next, one of the things on the roadmap is the ability for users to archive api keys (to disable them, but with a recovery path), that's a feature I hope to ship not long after.

So tl;dr: in the future this will be possible for our users to do it themselves, but that is not possible yet.

**Comment by BenjaminVettori** [comment]
Thank you, than answers my question 👍
